.github/ISSUE_TEMPLATE/new_feature.yml

```diff
@@ -1,53 +1,37 @@
 name: ✨ New Feature
-assignees:
-  - KRTirtho
 description: Suggest an idea
-title: "[Feature] <title>"
```

Comment:
Why remove this? Ok, just replace the `<title>` with `<your-issue-title>` part

Reply:
Because some people didn't understand it and it doesn't look nice

Reply:
That's just some people. Not everyone...
That's why, `your-issue-title` or `write-title-here` etc will help